components/common/shoppingCart/ShoppingCartItemRow.vue

```diff
@@ -2,33 +2,39 @@
   <div
     ref="hoverRef"
     class="is-flex is-justify-content-space-between background-hover">
-    <div class="is-flex pr-2">
-      <nuxt-link
-        :to="`/${urlPrefix}/gallery/${nft.id}`"
-        @click.native="emit('click-item')">
-        <BasicImage
-          :src="avatar"
-          :alt="nft?.name"
-          class="border image is-48x48" />
-      </nuxt-link>
-
-      <div
-        class="is-flex is-flex-direction-column is-justify-content-space-between ml-4 limit-width">
+    <div class="pr-2 w-full no-wrap is-clipped ellipsis">
+      <div class="is-flex">
         <nuxt-link
           :to="`/${urlPrefix}/gallery/${nft.id}`"
-          class="has-text-weight-bold has-text-color line-height-1 no-wrap is-clipped ellipsis"
           @click.native="emit('click-item')">
-          {{ nft.name }}
+          <BasicImage
+            :src="avatar"
+            :alt="nft?.name"
+            class="border image is-48x48" />
         </nuxt-link>
-        <div class="line-height-1 no-wrap is-clipped ellipsis">
-          {{ nft.collection?.name || nft.collection.id }}
+        <div
+          class="is-flex is-flex-direction-column is-justify-content-space-between ml-4 min-width w-full line-height-1">
+          <nuxt-link
+            :to="`/${urlPrefix}/gallery/${nft.id}`"
+            class="has-text-weight-bold has-text-color no-wrap is-clipped ellipsis"
+            @click.native="emit('click-item')">
+            {{ nft.name }}
+          </nuxt-link>
+          <div class="is-flex is-justify-content-space-between w-full">
+            <div class="no-wrap is-clipped ellipsis">
+              {{ nft.collection?.name || nft.collection.id }}
+            </div>
+
+            <div :class="[{ hidden: isHovered }, 'ml-2']">
+              <CommonTokenMoney :value="nft.price" :round="2" />
+            </div>
+          </div>
         </div>
       </div>
     </div>
-
     <div
       v-if="isHovered"
-      class="is-flex is-justify-content-end is-align-items-center">
+      class="is-flex is-justify-content-end is-align-items-center h-full m-auto ml-2">
```

Comment:
What is the `m-auto`? Does it make something different?

Reply:
Apparently, it's a stale class.
It was previously used to center the trash can but `align-self-center` does it better.